db/migrate/20160907144809_add_delivery_status_to_mail_server_logs.rb

```diff
@@ -0,0 +1,9 @@
+class AddDeliveryStatusToMailServerLogs < ActiveRecord::Migration
+  def up
+    add_column :mail_server_logs, :delivery_status, :string, :null => true
```

Comment:
good point, think I forced the issue to reset it from where I'd temporarily made it default to a blank string